airbyte-integrations/connectors/source-datascope/source_datascope/spec.json

```
@@ -0,0 +1,32 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Datascope Spec",
+    "type": "object",
+    "required": ["api_key", "form_id"],
+    "additionalProperties": false,
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "description": "api Key",
+        "airbyte_secret": true
+      },
+      "start_date": {
+        "type": "string",
+        "description": "Start date",
+	"pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
+        "examples": ["2017-01-25T00:00:00Z"]
+      },
+      "form_id": {
+        "type": "string",
+        "description": "Form ID"
+      },
+      "schema_type": {
+        "type": "string",
+        "description": "Please select type of schema 'fixed' or 'dynamic'",
```

this can be an enum which makes it a dropdown in the UI
```suggestion
        "description": "Please select type of schema 'fixed' or 'dynamic'",
        "enum": ["fixed", "dynamic"],
```